### ISSUE TITLE
Unknown Permission Issues 

### ISSUE THREAD

**Issue by embcla** [rancher/local-path-provisioner#79]
Hi guys

I've been using the lpp for some time, then I deicided to try something new.

Use case: edge cluster with shared folders
Status: in vm simulation, using vbox shared folders

Configuration: lpp cm updated to point to shared folder /persistentvolume instead of /var/lib/rancher/k3s/storage

Issue: nginx can read files from the persisten volume configured on the shared folder without issues, prometheus can't write in the folder and crashes

The only real debug I can get is the error log of prometheus which says only "I can't write so I'm panicking"
Full debug here
https://rancher-users.slack.com/archives/CGGQEHPPW/p1583445098193200?thread_ts=1583445098.193200&cid=CGGQEHPPW

Suggestion: additional documentation, helpful info, logs, configs about permissions necessary for lpp to work

**Comment by yasker** [comment]
Prometheus used to have some permission issue, see https://github.com/coreos/prometheus-operator/issues/541 .

Does it work if you're not using the vbox shared folder?

**Comment by embcla** [comment]
Hi @yasker 

I do not use the prometheus operator, rather the stable/prometheus helm chart.
And yes: Prometheus works just fine with the lpp configmap untouched. But that means it creates a random volume on a random node at initial startup.

Thanks for the pointer to prometheus issues, to be fair I did not specify the storage class, I changed the default configmap for K3S' local-path-provisioner deployment
So I can't follow the Prometheus' issue advice
